iiif_prezi3/helpers/make_collection.py

```diff
@@ -1,17 +1,23 @@
 from ..loader import monkeypatch_schema
-from ..skeleton import Collection
+from ..skeleton import Collection, CollectionRef
 
 
 class MakeCollection:
 
     def make_collection(self, **kwargs):
-        """Create a Collection.
+        """Creates a Collection or CollectionRef.
 
         Creates a new collection, adds it to the calling Collection `items`
         and returns the newly created object. Accepts keyword arguments to
         customize the resulting instance.
         """
```

Comment:
We should update the description to mention if you don't pass in `items` then it creates a CollectionRef.

How easy is it to convert a CollectionRef to Collection? If a user doesn't pass items then tries to add it later will this cause an error? Should we have two methods, make_collection and make_collection_ref to make it more explicit?